### PR TITLE
fix(osc): compatible with cloud odc code, fix ScheduleTaskParameter deserialize…

### PR DESCRIPTION
**Pull request by LioRoger**
… failed for osc

#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
1. fix OnlineSchemaChangeParameters deserialize to ScheduleTaskParameters failed
2. refactor some osc code to compatible with odc cloud code

#### Which issue(s) this PR fixes:

Fixes #
1. fix OnlineSchemaChangeParameters deserialize to ScheduleTaskParameters failed
2. refactor some osc code to compatible with odc cloud code
#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yhilmare — APPROVED** [review]
lgtm

File: server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oscfms/OscActionFsmBase.java [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+     */
+    protected boolean isTaskExpired(OscActionContext context) {
+        ScheduleTaskEntity scheduleTask = context.getScheduleTask();
+        OnlineSchemaChangeScheduleTaskParameters parameters =
+                parseOnlineSchemaChangeScheduleTaskParameters(scheduleTask.getParametersJson());
+        // check task is expired
+        Long scheduleId = context.getSchedule().getId();
+        Long scheduleTaskId = scheduleTask.getId();
+        Duration between = Duration.between(scheduleTask.getCreateTime().toInstant(), Instant.now());
+        log.info("Schedule id {} to check schedule task status with schedule task id {}", scheduleId, scheduleTaskId);
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oscfms/OscActionFsmBase.java:203]: odc use key=value log style.

e.g. `log.info("Check schedule task status, scheduleId={}, scheduleTaskId={}", scheduleId, scheduleTaskId)`

key=value style is easy for log analyeis by machine.

so as below.

Comment (LioRoger) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/oscfms/OscActionFsmBase.java:203]: it's a code migrate, not contains logic change. i will create another PR to fix this type issue

**Review by guowl3 — APPROVED** [review]
lgtm

**Review by yizhouxw — APPROVED** [review]

**Review by LuckyPickleZZ — APPROVED** [review]

**Pull request merged**